spec/order_spec.rb

```diff
@@ -2,30 +2,34 @@
 
 describe Order do
   TEST_PRICE = 8
```

Comment:
Nice, I now see why the Test price was at the beginning of the described block as it's being reused a couple of times!